spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/JedisConnectionConfiguration.java

```diff
@@ -95,6 +96,18 @@ private JedisClientConfigurationBuilder applyProperties(JedisClientConfiguration
 		return builder;
 	}
 
+	private void createPoolingBuilder(JedisClientConfigurationBuilder builder, Environment environment, Pool pool) {
+		final boolean poolEnabled = environment.getProperty("spring.redis.jedis.pool.enabled", Boolean.class, true);
+		if (poolEnabled) {
+			applyPooling(pool, builder);
+		}
+		final boolean isSentinelConfig = (getSentinelConfig() != null);
+		// Jedis Sentinel cannot operate without a pool.
+		if (!poolEnabled && isSentinelConfig) {
+			throw new RedisClientPoolingException(ClientType.JEDIS);
+		}
+	}
+
```

Comment:
Wouldn't it be better to add `return;` after line 102 and remove the `!poolEnabled` check on line 106?

Reply:
I done it. Thank you for reviewing the code, you can see it in the latest commit